nat-lab/tests/helpers.py

```diff
@@ -310,6 +313,9 @@ async def setup_environment(
         ),
     )
 
+    if prepare_vpn:
```

Comment:
Maybe move this before `setup_clients`, just in case somebody will add wg_server as peer

Reply:
This makes it slightly more complicated, as current VPN configuration is refering to the list of the nodes in API to configure required public keys to be accepted in VPN containers.

But `.setup_client`s establishes this list.

Reply:
`setup_api` does that, `setup_clients` already uses created meshmap, so it would be appropriate to run `prepare_vpn` before `setup_clients`. Unless I am not getting the idea what you are saying :smile:

Reply:
oh yeah, you are right. Fixed.